receiver/githubreceiver/internal/scraper/githubscraper/schema.graphql

```diff
@@ -1443,7 +1443,7 @@ type CommitComment implements Node & Comment & Deletable & Minimizable & Updatab
   viewerCanReact: Boolean!
   "Check if the current viewer can update this object."
   viewerCanUpdate: Boolean!
-  "Reasons why the current viewer can not update this comment."
+  "Reasons why the current viewer cannot update this comment."
```

Comment:
I think that schema is generated, might be best to not update directly. @adrielp fyi

Reply:
Hmm, indeed, it's over here: https://github.com/github/docs/blob/7c0a4f3519947036f0831c4d37e616a716c7f4a4/content/graphql/guides/using-the-graphql-api-for-discussions.md?plain=1#L647

I think I gave up trying to get github to fix that stuff... I'll drop when I get around to splitting.